resources/build-npm.js

```diff
@@ -13,6 +13,14 @@ const {
   showDirStats,
 } = require('./utils.js');
 
+const entryPoints = [
+  'index.ts',
+  'execution/execute.ts',
+  'jsutils/instanceOf.ts',
+  'language/parser.ts',
+  'language/ast.ts',
+];
```

Comment:
This is a random selection of exports mentioned over in https://github.com/graphql/graphql-js/issues/4074
There's no real logic which ones I chose here.